src/commons/sagas/RequestsSaga.ts

```diff
@@ -429,8 +429,10 @@ export const getAssessmentOverviews = async (
   const assessmentOverviews = await resp.json();
 
   return assessmentOverviews.map((overview: any) => {
-    overview.gradingStatus = computeGradingStatus(
+    // TODO: remove and replace with just isGradingPublished.
```

Comment:
Is this TODO still relevant?

Reply:
yep, I forget to remove the frontend field calculation even though it is no longer needed for the profile display... Removed it now!